components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -308,10 +308,10 @@ public static OMElement buildOMElement(InputStream inputStream) throws Exception
 
     }
 
-    public List<AppRegisteredUsersDTO> getAppRegisteredUsers(String subscriberName) throws APIMgtUsageQueryServiceClientException {
+    public List<AppRegisteredUsersDTO> getAppRegisteredUsers(String subscriberName, String groupId) throws APIMgtUsageQueryServiceClientException {
 
```

Comment:
@sinthuja this changes are moved to
 org.wso2.carbon.apimgt.usage.client.impl.APIUsageStatisticsRdbmsClientImpl.java
org.wso2.carbon.apimgt.usage.client.impl.APIUsageStatisticsRestClientImpl.java classes. hence fixed in there in commit: https://github.com/wso2/carbon-apimgt/commit/ef30cea43f77c9b6f6500f3f70449d7238a942e4